sys/include/stdio_uart.h

```diff
@@ -19,6 +19,34 @@
  * USEMODULE += stdin
  * ```
  *
+ * @attention   Using STDIO over UART from interrupt context should be avoided,
+ *              except for debugging purposes
+ *
+ * For testing purposes and using STDIO within an ISR should mostly work good
+ * enough and for some platforms even reliable. Production code however should
+ * fully avoid any access to STDIO from interrupt context. Instead, e.g. an
+ * event could be posted to an @ref sys_event and the actual STDIO operation
+ * being deferred to thread context.
+ *
+ * Some reasons why STDIO over UART from ISR should be avoided:
+ * 1. UART is *slow* and the system easily remains in interrupt context for
+ *    unacceptable long time.
+ *     - E.g. sending 100 chars at 9600 baud will block the system for
+ *       10 milli seconds.
```

Comment:
```suggestion
 *       100 milli seconds.
```

9600 baud are ~1000b/sec...

Reply:
maybe remove the space also, millisecond is a word ;)

Reply:
9,600 is closer to 10,000 than to 1,000 ;-)

100 / 9600 baud = 0.01042s ~= 10ms

Reply:
yeah, it is closer to 10000, but a baud with only two symbols (0 and 1) is closer to bit than to char... (100 * 8) / 9600 = 0.0833333 ~= 83ms.

Reply:
I always assumed the whole 8 data bits + parity and stop bits would count as a symbol :-/

Fixed. Thanks!